### ISSUE TITLE
CMD-F does not open filters, it opens search

### ISSUE THREAD

**Issue by tdegrunt** [TablePlus/TablePlus#1518]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Not relevant
2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
272
3. The steps to reproduce this issue:
Press CMD-F it opens search of sql query pane, not filters.

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
Hi @tdegrunt TablePlus does not have a filter in the query editor, which is used for unidentified queries. You can only open the advanced filter in the table view.
We may add the search within the result in the future release.
This can be a duplicated with https://github.com/TablePlus/TablePlus/issues/530

**Comment by huyphams** [comment]
hmm sorry, the tooltip was wrong, Command + F is only for show the filter and also focus the text field.
If you want to hide, press the ESC. There is a tutorial below the filter.

**Comment by huyphams** [comment]
Yep, that one is the filter, you can filter the content with it, TablePlus will generate the query based on that filter.
This is not the Search feature, it will generate a query with WHERE condition. You can choose the column for filtering in the dropdown Raw SQL.

**Comment by huyphams** [comment]
here is its documents: https://docs.tableplus.com/gui-tools/filter

**Comment by tdegrunt** [comment]
Ok, showing the filter is what I expected when I press cmd-F in the table view. Instead it opens the SQL query pane. The docs say it would open the filter, but it does not seem like it does?

**Comment by huyphams** [comment]
that is not a query pane, that is the filter. I still don't understand your issue
![2019-10-29 16 26 00](https://user-images.githubusercontent.com/3654622/67801866-d9da0e80-fa68-11e9-9dc1-5036ac205a84.gif)

**Comment by huyphams** [comment]
The Raw SQL means you can query with any column with auto completion, please see the gif:
![2019-10-29 16 26 00](https://user-images.githubusercontent.com/3654622/67801952-1148bb00-fa69-11e9-9dea-f6a912972062.gif)
You can change the behavior with the settings:
<img width="761" alt="Screen Shot 2019-10-29 at 4 28 33 PM" src="https://user-images.githubusercontent.com/3654622/67802006-27ef1200-fa69-11e9-9fc3-be4283345bcb.png">

**Comment by tdegrunt** [comment]
Thanks for your quick responses! I made a screen recording, but can't upload it to GitHub, this is the bad-version animated GIF, sorry:

![cmdf](https://user-images.githubusercontent.com/46554/67806375-09057580-fa93-11e9-8254-2701efe0c13d.gif)

**Comment by huyphams** [comment]
Hey @tdegrunt could you try again with the latest release and see how's it going?

**Comment by tdegrunt** [comment]
Hello @huyphams, same behaviour. CMD-F does not trigger the filters, it switches to the SQL query (same as CMD-enter).

**Comment by huyphams** [comment]
Hi @tdegrunt please try again with our latest release 342 and see how it goes

**Comment by tdegrunt** [comment]
Thank you! Finally I think I understand what is going on. I'm using the [Colemak Keyboard layout](https://colemak.com) and when I press the physical F key, that means I see a 'T' on screen. My F key is physically the E key. When I pressed the physical F key, the filters popped up, when I'm in the table view. None of the other keyboard shortcuts have this problem though.
